src/components/two-up-display.tsx

```diff
@@ -90,11 +92,41 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
         <div className={contentClass} data-test="right-content">
           {this.props.rightPanel}
         </div>
-        {/* <div className={"footer " + this.props.spaceClass} data-test="right-footer"/> // removed for time being*/}
+        {this.renderRightPanelFooter()}
       </div>
     );
   }
 
+  private renderRightPanelFooter(){
+    if (this.props.spaceClass === "populations" &&
+        this.props.selectedRightPanel === "data" &&
+        this.props.rightPanelButtons) {
+      const buttons = this.props.rightPanelButtons.map( (button: any) => {
+        const type = button.type || "button";
+        let title = button.title;
+        if (button.title === "Scale") {
+          title = button.value ? "Show Recent Data" : "Show All Data";
+        }
+        if (type === "button") {
+          return (
+            <div key={button.title} className="button-holder" onClick={button.action}>
+              {title}
+            </div>
+          );
+        }
+      });
+
+      return (
+        <div className={"footer " + this.props.spaceClass} data-test="right-footer">
+          { buttons }
+        </div>
+      );
+    } else {
+      return null;
+    }
+
```

Comment:
Not sure if the button is going to eventually live in the footer, but we will need this space for other graph buttons eventually.  This button might end up floating on top of the graph in the upper right corner.